apps/hash-frontend/src/components/hooks/use-logout-flow.ts

```diff
@@ -32,9 +35,11 @@ export const useLogoutFlow = (deps?: DependencyList) => {
       if (logoutToken) {
         await oryKratosClient.updateLogoutFlow({ token: logoutToken });
 
+        await refetchUser();
+
         resetLocalStorage();
 
-        await client.resetStore();
+        await client.clearStore();
```

Comment:
@CiaranMn @nonparibus when a user logs out, currently they stay on the current page they're on. That means that if they're on a page, and logout, they'll see the page in readonly mode (tried to showcase that in the demo video in the PR description).

Are you okay with this behaviour for now, or would you like me to redirect the user to the `/login` page on logout?

Reply:
I'm fine with this for the timebeing. Thanks.